cli/cmd/aeon.go

```diff
@@ -42,14 +49,16 @@ tt aeon connect unix://<socket-path>`,
 				internalAeonConnect, args)
 			util.HandleCmdErr(cmd, err)
 		},
-		Args: cobra.ExactArgs(1),
 	}
+
 	aeonCmd.Flags().StringVar(&connectCtx.Ssl.KeyFile, "sslkeyfile", "",
 		"path to a private SSL key file")
 	aeonCmd.Flags().StringVar(&connectCtx.Ssl.CertFile, "sslcertfile", "",
 		"path to a SSL certificate file")
 	aeonCmd.Flags().StringVar(&connectCtx.Ssl.CaFile, "sslcafile", "",
 		"path to a trusted certificate authorities (CA) file")
+	aeonCmd.Flags().StringVarP(&configPath, "config", "c", "", "path config")
+	aeonCmd.Flags().StringVarP(&instance, "instanceName", "i", "", "instance name")
```

Comment:
Let's better make such interface:
```
tt aeon connect path/to/config.yml instance_name
```
As example:
```
tt aeon connect app/config.yml instance-001
```

It would be much better than:

```
tt aeon connect -c app/config.yml -i instance-001
```

Reply:
Done